app/services/gtfs_redis_adapter.rb

```diff
@@ -57,6 +70,13 @@ def set_gtfs_stop_times(stop_id, stop_times, data_provider_id)
     redis.rpush("#{namespace}:gtfs:#{data_provider_id}:stop:#{stop_id}:stop_times", stop_times.to_json)
   end
 
+  # delete all key starting with stop
+  def delete_all_stop_times(data_provider_id)
+    redis.scan_each(match: "#{namespace}:gtfs:#{data_provider_id}:stop:*") do |key|
```

Comment:
`stop_times` instead of `stop`?

Reply:
the redis key is "#{namespace}:gtfs:#{data_provider_id}:stop:#{stop_id}:stop_times"

so we by deleting "#{namespace}:gtfs:#{data_provider_id}:stop:*"
we delete all "#{stop_id}:stop_times" => all stop_times for all stops

the stops itself are stored in SQL DB and not in redis

Reply:
ok